gcp-connector-util/init.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/google/cloud-print-connector/gcp"
 	"github.com/google/cloud-print-connector/lib"
 	"github.com/urfave/cli"
+  "github.com/satori/go.uuid"
```

Comment:
Can you run `go fmt` on this change? It looks like the spacing/ordering is not right.